spec/music_album_spec.rb

```diff
@@ -0,0 +1,17 @@
+require './ruby_classes/music_album'
+
+describe MusicAlbum do
+  context 'When MusicAlbum is on_spotity to true' do
+    music_album_one = MusicAlbum.new('name', Time.new(11 / 15 / 2022), true)
+    it 'Should return an instance of the Class musicAlbum' do
+      expect(music_album_one).to be_an_instance_of MusicAlbum
+    end
+  end
+
+  context 'When MusicAlbum is on_spotity to false' do
+    music_album_two = MusicAlbum.new('name', Time.new(11 / 15 / 2022), false)
+    it 'Should return an instance of the Class musicAlbum' do
+      expect(music_album_two).to be_an_instance_of MusicAlbum
+    end
+  end
+end
```

Comment:
It would be nice if you add an extra case for each test case (music_album_one and music_album_two) in order to make sure that you're setting the info in the right way, you could do it in that way:

![image](https://user-images.githubusercontent.com/97189760/202007034-573dd38c-a8c0-4891-b177-9282f4d09245.png)


This would let the reviewer realize that you're not only returning instances of the class (which is ok) it will also show that you are setting the values in the `on_spotify` exactly how you want to set them.

Reply:
Ok bro, I will add...